coins.json

```diff
@@ -602,5 +602,21 @@
         "publicKeyType": "ed25519",
         "transactionPath": "/transaction/",
         "explorer": "https://semux.info/explorer"
+    },
+    {
+        "id": "monetaryunit",
+        "name": "MonetaryUnit",
+        "symbol": "MUE",
+        "decimals": 8,
+        "blockchain": "Bitcoin",
+        "derivationPath": "m/44'/31'/0'/0/0",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1",
+        "publicKeyHasher": "sha256ripemd",
+        "base58Hasher": "sha256d",
+        "xpub": "xpub",
+        "xprv": "xprv",
+        "transactionPath": "/tx/",
+        "explorer": "https://blockbook.monetaryunit.org"
```

Comment:
looks like https://explorer.monetaryunit.org/ is official block explorer? let's use it